src/main/java/org/opentripplanner/updater/bike_rental/datasources/params/BikeRentalDataSourceParameters.java

```diff
@@ -42,4 +49,8 @@ public String getNetwork(String defaultValue) {
   public String getApiKey() {
     return apiKey;
   }
+
+  public Map<String, String> getHttpHeaders() {
```

Comment:
I think this should be `@Nonnull` so we do not have to check for null later. Add `@Nonnull` to the constructor parameter as well.